fili-core/src/test/resources/testApplicationConfig.properties

```diff
@@ -51,6 +51,13 @@ bard__druid_dim_loader_timer_delay = 60000
 # Data Cache V2 (needs the above flag set as well)
 bard__druid_cache_v2_enabled = false
 
+# Caching strategy, which is one of the following
+# 1. ETag
+# 2. LocalSignature (cache V2)
+# 3. TtlOnly (cache V1)
+# 4. None (no caching)
+druid__query_response_caching_strategy = None
+
```

Comment:
We discussed if caching strategy is set, then cache_enabled and cache_v2_enabled would be removed.